### ISSUE TITLE
Sort by Tags and Authors

### ISSUE THREAD

**Issue by mtorials** [mtorials/hugo-mt32#5]
implement a function to search for specific tags and authors!

**Comment by ShowMeYourSkil** [comment]
The reader should can click the tags in the readme. Now he can see all articels with the tag.

Example:
#automation

1. Ansible install
2. Ansible
3. Ansible

**Comment by mtorials** [comment]
Yes this is how I wanted it to work. Should be quite doable with hugo.
